minidump-processor/src/stackwalker/amd64.rs

```diff
@@ -281,13 +281,13 @@ where
             // It's unclear why, perhaps the test is buggy, but for now we
             // preserve that behaviour.
             if let Some(last_bp) = last_bp {
-                let address_of_bp = address_of_ip - POINTER_WIDTH;
+                let address_of_bp = address_of_ip.checked_sub(POINTER_WIDTH)?;
                 // Can assume this resolves because we already walked over it when
                 // checking address_of_ip values.
-                let bp = stack_memory.get_memory_at_address(address_of_bp as u64)?;
+                let bp: u64 = stack_memory.get_memory_at_address(address_of_bp as u64)?;
                 if last_bp == address_of_bp
                     && bp > address_of_ip
-                    && bp - address_of_bp <= MAX_REASONABLE_GAP_BETWEEN_FRAMES
+                    && bp.checked_sub(address_of_bp)? <= MAX_REASONABLE_GAP_BETWEEN_FRAMES
```

Comment:
This is definitely hard to see but I believe this underflow cannot happen.

If we reach this point in this code then we know:

```
address_of_bp < address_of_ip < bp
              ^               ^
     by construction          |
                              |
                immediately preceding check
```
So `bp - address_of_bp` is always positive.